pkg/email/summary/reporter.go

```diff
@@ -24,19 +25,20 @@ func (o *Reporter) Report(sources []Source, format string) (email.Report, error)
 	err = templ.Execute(b, struct {
 		Sources     []Source
 		ClusterName string
-	}{Sources: sources, ClusterName: o.clusterName})
+		TitlePrefix string
+	}{Sources: sources, ClusterName: o.clusterName, TitlePrefix: o.titlePrefix})
 	if err != nil {
 		return email.Report{}, err
 	}
 
 	return email.Report{
 		ClusterName: o.clusterName,
-		Title:       "Summary Report from " + time.Now().Format("2006-01-02"),
+		Title:       o.titlePrefix + " (summary) on " + o.clusterName + " from " + time.Now().Format("2006-01-02"),
```

Comment:
Because the clusterName is optional and not always set, the title should reflect this and only include the clustername when it’s set.

Reply:
Yes, thanks

Reply:
Just a suggestion, not pushed but already merged.